homeassistant/components/zha/strings.json

```diff
@@ -502,5 +504,15 @@
         }
       }
     }
+  },
+  "issues": {
+    "wrong_silabs_firmware_installed_nabucasa": {
+      "title": "Zigbee radio has the wrong firmware installed",
```

Comment:
This message will be shown if *any* wrong firmware is detected (e.g. the stick is stuck in the bootloader because a firmware upgrade was interrupted, it has OpenThread firmware, ...) so I wanted to keep it generic.